content/articles/getting-started-ssl-certificates.markdown

```diff
@@ -20,14 +20,14 @@ Getting a new SSL certificate is a multi-step process and involves several parti
 
 ## SSL certificate requirements {#requirements}
 
-Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration or a feature to install a custom certificate on your server.
+Before starting, take a moment to review how you will use the certificate and be sure to have access to the server configuration, or a feature to install a custom certificate on your server.
```

Comment:
It depends.

We expect you to know what a certificate is for, as you came to us with the need of a certificate (hence you probably expect to use it for HTTPS).

The sentence essentially tells you "be sure to understand your needs before starting" and also "we ask to check if you have access to the server configuration".

This latter information depends on your infrastructure.